src/renderer/Status.tsx

```diff
@@ -3,7 +3,24 @@ import eyeIcon from  '../../assets/icon/sleep-icon.png';
 import errorIcon from  '../../assets/icon/error-icon.png';
 import watchIcon from  '../../assets/icon/watch-icon.png';
 import savingIcon from '../../assets/icon/saving-icon.png';
-import * as React from 'react';
+import { useState, useEffect } from 'react';
+import { AppStatus } from 'main/types';
+
+type StatusMessageType = 'error' | 'status';
+
+type StatusMessageObjectType = {
+  title: string,
+  type: StatusMessageType,
+  icon: string,
+};
+
+const statusMessages: { [key: number]: StatusMessageObjectType } = {
+  [AppStatus.WaitingForWoW]:   { type: 'status', icon: eyeIcon, title: 'Waiting for WoW to start' },
+  [AppStatus.Recording]:       { type: 'status', icon: recordIcon, title: 'Recording' },
+  [AppStatus.InvalidConfig]:   { type: 'error', icon: errorIcon, title: 'Failed to launch, check config is valid' },
+  [AppStatus.ReadyAndWaiting]: { type: 'status', icon: watchIcon, title: 'Ready and waiting'},
```

Comment:
ReadyAndWaiting could be better named? We're buffering recording files at this point so doing work, not quite what is implied by this name

Reply:
I just named them more or less according to how they were described :D I'll find a better name. I initially wanted to go with `ReadyToRecord`.